### PR TITLE
[ISSUE#7344] subscruber ITCase runs failed with IlleagalArgumentException when use httpClientProxy only

### LINKED ISSUE #7344: Run subscruber ITCase failed when use httpClientProxy only.
**Describe the bug**
Disable `grpcClientProxy`  and use `httpClientProxy` only in `NamingClientProxyDelegate`, then run `Subscribe_ITCase`，`SubscribeCluster_ITCase` failed.

**Expected behavior**
should run properly successfully.

**Acutally behavior**
1. run Subscribe_ITCase.java, result as below:
```
java.lang.IllegalArgumentException: Param 'groupName' is illegal, groupName is blank
	at com.alibaba.nacos.api.naming.utils.NamingUtils.getGroupedName(NamingUtils.java:50)
	at com.alibaba.nacos.client.naming.event.InstancesChangeNotifier.onEvent(InstancesChangeNotifier.java:114)
	at com.alibaba.nacos.client.naming.event.InstancesChangeNotifier.onEvent(InstancesChangeNotifier.java:40)
	at com.alibaba.nacos.common.notify.DefaultPublisher.lambda$notifySubscriber$0(DefaultPublisher.java:201)
	at com.alibaba.nacos.common.notify.DefaultPublisher.notifySubscriber(DefaultPublisher.java:208)
	at com.alibaba.nacos.common.notify.DefaultPublisher.receiveEvent(DefaultPublisher.java:192)
	at com.alibaba.nacos.common.notify.DefaultPublisher.openEventHandler(DefaultPublisher.java:118)
	at com.alibaba.nacos.common.notify.DefaultPublisher.run(DefaultPublisher.java:95)
```
2. SubscribeCluster_ITCase.java Tests failed: 2, passed: 2 of 4 tests - 38 s 406 ms
```
java.lang.IllegalArgumentException: Param 'groupName' is illegal, groupName is blank
	at com.alibaba.nacos.api.naming.utils.NamingUtils.getGroupedName(NamingUtils.java:50) ~[classes/:na]
	at com.alibaba.nacos.client.naming.event.InstancesChangeNotifier.onEvent(InstancesChangeNotifier.java:114) ~[classes/:na]
	at com.alibaba.nacos.client.naming.event.InstancesChangeNotifier.onEvent(InstancesChangeNotifier.java:40) ~[classes/:na]
	at com.alibaba.nacos.common.notify.DefaultPublisher.lambda$notifySubscriber$0(DefaultPublisher.java:201) [classes/:na]
	at com.alibaba.nacos.common.notify.DefaultPublisher.notifySubscriber(DefaultPublisher.java:208) [classes/:na]
	at com.alibaba.nacos.common.notify.DefaultPublisher.receiveEvent(DefaultPublisher.java:192) [classes/:na]
	at com.alibaba.nacos.common.notify.DefaultPublisher.openEventHandler(DefaultPublisher.java:118) [classes/:na]
	at com.alibaba.nacos.common.notify.DefaultPublisher.run(DefaultPublisher.java:95) [classes/:na]
```

**How to Reproduce**
1. Go to 'NamingClientProxyDelegate.java'
2.  make method `NamingClientProxyDelegate#getExecuteClientProxy` always return `httpClientProxy`, like  
```
    private NamingClientProxy getExecuteClientProxy(Instance instance) {
        return httpClientProxy;
    }
```
3. remove all `grpcClientProxy` declaration and replace `grpcClientProxy` invocation, like
```
getExecuteClientProxy(null).subscribe(serviceName, groupName, clusters)
```
4. run subscriber ITCase and see error

**Desktop (please complete the following information):**
 - OS: windows 10
 - Version: nacos-server 2.1.0-SNAPSHOT, nacos-client 2.1.0-SNAPSHOT
 - Module:  naming

### PR DESCRIPTION
**Pull request by lpaz010**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

Fixes #7344 

## Brief changelog

Modify `NamingUtils#getGroupedName(final String serviceName, final String groupName)`,  if `serviceName` has combined like '_groupName_A@@serviceName_A_' and `groupName` is nil, then return serviceName directly.

## Verifying this change

run `Subscribe_ITCase` and `SubscribeCluster_ITCase` with `HttpClientProxy` only.

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [ ] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

**Comment by lpaz010** [comment]
可以升级JUnit版本高于4.12嘛

File: api/src/main/java/com/alibaba/nacos/api/naming/utils/NamingUtils.java [hunk 1]
```diff
@@ -47,7 +49,9 @@ public static String getGroupedName(final String serviceName, final String group
             throw new IllegalArgumentException("Param 'serviceName' is illegal, serviceName is blank");
         }
         if (StringUtils.isBlank(groupName)) {
-            throw new IllegalArgumentException("Param 'groupName' is illegal, groupName is blank");
+            // fix #7344
```
Comment (realJackSun) [inline api/src/main/java/com/alibaba/nacos/api/naming/utils/NamingUtils.java:52]: 集成测试不过最好还是直接修改测试类吧，不要直接修改代码

**Review by realJackSun — CHANGES_REQUESTED** [review]

File: api/src/main/java/com/alibaba/nacos/api/naming/utils/NamingUtils.java (hunk 1 above)
Comment (MajorHe1) [reply api/src/main/java/com/alibaba/nacos/api/naming/utils/NamingUtils.java:52]: 这个我觉得不是集成测试的问题，就是1.x版本和2.x版本推送的逻辑实现不一样，一个把group跟name拼接起来了，一个没有拼接。虽然正常使用没有出现问题，因为2.x的客户端都是长连接推送，但毕竟问题还是存在的，在客户端多用一个判断来兼容感觉无伤大雅。

**Comment by KomachiSion** [comment]
CLA not sign for a long time.

**Pull request closed without merging**